### ISSUE TITLE
Question: how to handle footnotes inside equations?

### ISSUE THREAD

**Issue by TimArborealis** [nlbdev/mathml-guidelines#45]
Sometimes equation annotations are lengthy or point to certain areas of the expression. Is there support for footnotes inside equations, or how should we advice to mark up stuff like this?

![bild](https://github.com/user-attachments/assets/0fa303d7-7359-47ac-aee8-ff6bf0e4f8fe)

**Comment by TimArborealis** [comment]
Looks like footnotes inside `<math>` , it won't validate. We need to advice for a less elegant solution until further notice.

Suggestion: Mark up the expression as image. Place MathML of expression (without footnote/annotation) in image description. Image description vendors can later complement the description with information about the comment and where it's pointing to.

**Comment by TimArborealis** [comment]
https://www.geeksforgeeks.org/html5-mathml-href-attribute/ Try out links inside math elements. Does it validate? Tim & Lisa investigate.

**Comment by MartheGjelstad** [comment]
I don't think we should use links inside math elements: https://developer.mozilla.org/en-US/docs/Web/MathML/Global_attributes/href

**Comment by samimaattaCelia** [comment]
Oh, seems like I accidentally wrote this section into the guidelines in commit 871287fadce626bd911d72b50dea5966b6223d9c. I guess I somehow mixed this up with #53, since it also talks about referencing footnotes.

**Comment by TimArborealis** [comment]
Closing this issue since Sami already wrote this section in accordance with what works (i.e. not links inside `<math>` elements).
